### ISSUE TITLE
Update repo README landing page

### ISSUE THREAD

**Issue by GeorgiaHCA** [alan-turing-institute/AutisticaCitizenScience#278]
## Summary

A lot has happened since the main README was created - it needs updating and also it should be clear and guide people through the process of contributing.

**Comment by GeorgiaHCA** [comment]
I've created an initial design for the page: https://github.com/alan-turing-institute/AutisticaCitizenScience/pull/281. It's still a work in progress and needs images, maybe diagrams, and an even simpler navigation.

**Comment by GeorgiaHCA** [comment]
Would you be able to review @JamesSCTJ ? @Angelsaur?

**Comment by GeorgiaHCA** [comment]
Thanks for your review @Angelsaur ! I've made the changes you recommended.

**Comment by GeorgiaHCA** [comment]
Thanks for your review @JamesSCTJ I've added your review to the community recommendations folder and made lots of your recommended changes. I left a few things in which I thought were important for emphasis. I think we definitely need a volunteer case study - possibly as a video of different volunteers talking about what they've done. Would you be interested in being in one or helping make it? All best Georgia. 

Really helpful! 

Best wishes, 

Georgia

**Comment by GeorgiaHCA** [comment]
Hey! I'd appreciate your review and thoughts Fujitsu team! The internal structure of the filing system and the links from the main page should fit with the work you're doing. Thanks so much!
